dm/syncer/syncer.go

```diff
@@ -913,11 +929,6 @@ func (s *Syncer) updateReplicationLagMetric() {
 			s.tctx.L().Info("ShowLagInLog", zap.Int64("lag", lag))
 		}
 	})
-
-	// reset skip job TS in case of skip job TS is never updated
-	if minTS == s.workerJobTSArray[skipJobIdx].Load() {
```

Comment:
why remove these lines

Reply:
skipped event shouldn't be accounted when calculating lag

and it's not used elsewhere

Reply:
but this do change `lag metric` behavior when syncer is catching up and the distance between up and downstream is quite long, and there's no interested binlog event in between.
before this pr, there's be a decreasing lag, but 0 after.

will revert it, should change it in another pr